app/views/layouts/_navbar.html.erb

```diff
@@ -11,6 +11,9 @@
       <li class='nav-item'>
       	<%= link_to 'Mi Perfil', edit_user_registration_path, class: 'nav-link' %>
       </li>
+      <li class='nav-item'>
+        <%= link_to 'Prueba Física', admin_physical_tests_new_path, class: 'nav-link' %>
```

Comment:
Instead of a `<li>` element here, let's add a helper called `admin_nav_links`. In that helper, check if `current_user.has_role :admin` and if true, return this list element (if we add more admin specific links in the future, we can add them to that helper as well). We need this because this link is showing up on the member navbar. 

Having said that, I feel like this link belongs in the Dashboard sidebar because that is the main navigation point. This is just my opinion, I'd like to know what everyone else thinks about this @adanvillarreal @javier96. I feel like everything should be under the dashboard layout to keep everything in the same place for easy access and a better UI/UX.